library/influxdb

```diff
@@ -1,15 +1,26 @@
-# maintainer: Shubhra Kar <shubhra@influxdb.com> (@ShubhraKar)
+Maintainers: Shubhra Kar <shubhra@influxdb.com> (@ShubhraKar)
 
-0.12: git://github.com/influxdata/influxdb-docker@6d869aa598baf9d23019682ecff42d022a00ce17 0.12
-0.12.2: git://github.com/influxdata/influxdb-docker@6d869aa598baf9d23019682ecff42d022a00ce17 0.12
+Tags: 0.12, 0.12.2
+GitRepo: git://github.com/influxdata/influxdb-docker
+GitCommit: 6d869aa598baf9d23019682ecff42d022a00ce17
+Directory: 0.12
 
-0.13: git://github.com/influxdata/influxdata-docker@215cf009c143dc739b5b10084ae330ca7f3665d6 influxdb/0.13
-0.13.0: git://github.com/influxdata/influxdata-docker@215cf009c143dc739b5b10084ae330ca7f3665d6 influxdb/0.13
-latest: git://github.com/influxdata/influxdata-docker@215cf009c143dc739b5b10084ae330ca7f3665d6 influxdb/0.13
+Tags: 0.13, 0.13.0, latest
+GitRepo: git://github.com/influxdata/influxdata-docker
+GitCommit: 5b975e66d1c36d87396256947db0c745b43844a1
```

Comment:
Is this the correct commit?  https://travis-ci.org/docker-library/official-images/builds/138948965#L446-L456